### DISCUSSION TITLE
Lesson 6: Brownie Fund Me - CompilerError: solc returned the following errors:

### DISCUSSION THREAD

**Discussion by I-M-F** [smartcontractkit/full-blockchain-solidity-course-py#69] [Q&A] [answered]
Hey there everyone. I'm really enjoying this course. journey to being an avid Smart Contract/Blockchain Software Engineer courtesy of @PatrickAlphaC and freecodecamp course.

Compiler Error : solc returned the following errors :
![image](https://user-images.githubusercontent.com/87519759/134131342-22b8e37e-710e-426f-9805-ec977597c69c.png)

INFO: Could not find files for the given pattern(s).
Brownie v1.16.3 - Python development framework for Ethereum

Compiling contracts...
  Solc version: 0.6.12
  Optimizer: Enabled  Runs: 200
  EVM Version: Istanbul        
CompilerError: solc returned the following errors:

contracts/FundMe.sol:5:1: ParserError: Source "smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol" not found: File not found.
import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
^--------------------------------------------------------------------------^

contracts/FundMe.sol:6:1: ParserError: Source "smartcontractkit/chainlink-brownie-contracts@1.1.1/contracts/src/v0.6/vendor/SafeMathChainlink.sol" not found: File not found.
import "@chainlink/contracts/src/v0.6/vendor/SafeMathChainlink.sol";
^------------------------------------------------------------------^ 

Brownie-config.yaml
![image](https://user-images.githubusercontent.com/87519759/134131536-4b9008b1-4b03-484f-923d-3fa3f7521e20.png)

dependancies: 
  # - <organisation/repo>@<version>
  - smartcontractkit/chainlink-brownie-contracts@1.1.1
compiler:
  solc:
    remappings:
      - '@chainlink=smartcontractkit/chainlink-brownie-contracts@1.1.1'

FundMe.sol 
Source "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol" not found: File import callback not supported
![image](https://user-images.githubusercontent.com/87519759/134131734-5fb01b83-4457-4ed6-8498-a9f4bf2ea139.png)

// SPDX-License-Identifier: MIT

pragma solidity ^0.6.6;

import "@chainlink/contracts/src/v0.6/interfaces/AggregatorV3Interface.sol";
import "@chainlink/contracts/src/v0.6/vendor/SafeMathChainlink.sol";

contract FundMe {
    using SafeMathChainlink for uint256;

    mapping(address => uint256) public addressToAmountFunded;
    address[] public funders;
    address public owner;

    constructor() public {
        owner = msg.sender;
    }

    function fund() public payable {
        uint256 minimumUSD = 50 * 10**18;
        require(
            getConversionRate(msg.value) >= minimumUSD,
            "You need to spend more ETH!"
        );
        addressToAmountFunded[msg.sender] += msg.value;
        funders.push(msg.sender);
    }

    function getVersion() public view returns (uint256) {
        AggregatorV3Interface priceFeed = AggregatorV3Interface(
            0x8A753747A1Fa494EC906cE90E9f37563A8AF630e
        );
        return priceFeed.version();
    }

    function getPrice() public view returns (uint256) {
        AggregatorV3Interface priceFeed = AggregatorV3Interface(
            0x8A753747A1Fa494EC906cE90E9f37563A8AF630e
        );
        (, int256 answer, , , ) = priceFeed.latestRoundData();
        return uint256(answer * 10000000000);
    }

    // 1000000000
    function getConversionRate(uint256 ethAmount)
        public
        view
        returns (uint256)
    {
        uint256 ethPrice = getPrice();
        uint256 ethAmountInUsd = (ethPrice * ethAmount) / 1000000000000000000;
        return ethAmountInUsd;
    }

    modifier onlyOwner() {
        require(msg.sender == owner);
        _;
    }

    function withdraw() public payable onlyOwner {
        msg.sender.transfer(address(this).balance);

        for (
            uint256 funderIndex = 0;
            funderIndex < funders.length;
            funderIndex++
        ) {
            address funder = funders[funderIndex];
            addressToAmountFunded[funder] = 0;
        }
        funders = new address[](0);
    }
}

Kindly assist how to resolve this error. Thank you in advance.

**Comment by I-M-F** [comment, marked as answer]
My apologies after a while...

I had a typo dependancies -> dependencies but it has a Warning should I be worried. 

INFO: Could not find files for the given pattern(s).
Brownie v1.16.3 - Python development framework for Ethereum

865kiB [00:02, 409kiB/s]  
WARNING: Unable to compile smartcontractkit/chainlink-brownie-contracts@1.1.1 due to a UnicodeDecodeError - you may still be able to import sources from the package, but will be unable to load the package directly.

Compiling contracts...
  Solc version: 0.6.12
  Optimizer: Enabled  Runs: 200
  EVM Version: Istanbul
Generating build data...
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/AggregatorV3Interface
 - smartcontractkit/chainlink-brownie-contracts@1.1.1/SafeMathChai
Project has been compiled.

**Reply by PatrickAlphaC** [reply]
Great work!
